api/src/main/java/io/strimzi/api/kafka/model/Http.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.sundr.builder.annotations.Buildable;
+import io.vertx.core.cli.annotations.DefaultValue;
+import lombok.EqualsAndHashCode;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * A representation of the configurable aspect of a HTTP.
```

Comment:
"... of a HTTP." what? It seems the description is cut.
Btw are you using this "simple" name because we think we can reuse this `Http` configuration for something different? So not using `KafkaBridgeHttpConfig` or something similar?